src/facade/dragonfly_connection.cc

```diff
@@ -1445,7 +1445,8 @@ auto Connection::IoLoop() -> variant<error_code, ParserStatus> {
           });
         }
 
-        DCHECK_GT(io_buf_.AppendLen(), 0U);
+        LOG_IF(WARNING, io_buf_.AppendLen() == 0U)
```

Comment:
please use LOG_EVERY_T

Reply:
oh, right - it's probably neededfor memcache?